testgrid/config/config.yaml

```diff
@@ -3681,30 +3681,30 @@ dashboards:
     test_group_name: ci-kubernetes-test-go
     base_options: 'include-filter-by-regex=kubectl%7COverall'
     description: 'unit, integration and cmd tests for master branch'
-  - name: skew-cluster1.7-kubectl1.8-gce
+  - name: skew-cluster-stable1-kubectl-latest-gce
     test_group_name: ci-kubernetes-e2e-gce-new-master-gci-kubectl-skew
-    description: '1.7 e2e tests run against a 1.7 gce cluster using a 1.8 kubectl binary'
-  - name: skew-cluster1.7-kubectl1.8-gke
+    description: 'stable1 e2e tests run against a stable1 gce cluster using a master kubectl binary'
+  - name: skew-cluster-stable1-kubectl-latest-gke
     test_group_name: ci-kubernetes-e2e-gke-new-master-gci-kubectl-skew
-    description: '1.7 e2e tests run against a 1.7 gke cluster using a 1.8 kubectl binary'
-  - name: skew-cluster1.8-kubectl1.7-gce
+    description: 'stable1 e2e tests run against a stable1 gke cluster using a master kubectl binary'
+  - name: skew-cluster-latest-kubectl-stable1-gce
     test_group_name: ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew
-    description: '1.7 e2e tests run against a 1.8 gce cluster using a 1.7 kubectl binary'
-  - name: skew-cluster1.8-kubectl1.7-gke
+    description: 'stable1 e2e tests run against a master gce cluster using a stable1 kubectl binary'
+  - name: skew-cluster-latest-kubectl-stable1-gke
     test_group_name: ci-kubernetes-e2e-gke-master-new-gci-kubectl-skew
-    description: '1.7 e2e tests run against a 1.8 gke cluster using a 1.7 kubectl binary'
-  - name: skew-cluster1.7-kubectl1.8-gce-serial
+    description: 'stable1 e2e tests run against a master gke cluster using a stable1 kubectl binary'
+  - name: skew-cluster-stable1-kubectl-latest-gce-serial
     test_group_name: ci-kubernetes-e2e-gce-new-master-gci-kubectl-skew-serial
-    description: '1.7 serial tests run against a 1.7 gke cluster using a 1.8 kubectl binary'
-  - name: skew-cluster1.7-kubectl1.8-gke-serial
+    description: 'stable1 serial tests run against a stable1 gke cluster using a master kubectl binary'
```

Comment:
gce cluster?

Reply:
good catch (it was wrong from the beginning)